vizro-core/docs/pages/user-guides/kedro-data-catalog.md

```diff
@@ -1,17 +1,16 @@
-# How to integrate Vizro with Kedro
+# How to integrate Vizro with Kedro Data Catalog
 
-This page describes how to integrate Vizro with [Kedro](https://docs.kedro.org/en/stable/index.html), an open-source Python framework to create reproducible, maintainable, and modular data science code. For Pandas datasets registered in a Kedro data catalog,
-Vizro provides a convenient way to visualize them.
+This page describes how to integrate Vizro with [Kedro](https://docs.kedro.org/en/stable/index.html), an open-source Python framework to create reproducible, maintainable, and modular data science code. For Pandas datasets registered in a Kedro data catalog, Vizro provides a convenient way to visualize them.
```

Comment:
Here we are referring to _Kedro_ datasets, which is what Kedro calls them so it's ok to use "dataset". This is actually one of the reasons I think we should ideally not refer to Vizro data as "datasets" because it might potentially cause confusion (?).

Reply:
Ahh okay - fine for me 😄